src/zui/components/ZUIVerticalBarChart/ZUIVerticalBarChartScale.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Typography } from '@mui/material';
+import React, { FC } from 'react';
+import { useTheme } from '@mui/system';
+
+export interface ZUIVerticalBarChartScaleProps {
+  maxValue: number;
+  minValue?: number;
+  orientation?: 'horizontal' | 'vertical';
+}
+const ZUIVerticalBarChartScale: FC<ZUIVerticalBarChartScaleProps> = ({
+  maxValue,
+  minValue = 0,
+  orientation = 'vertical',
+}) => {
+  const theme = useTheme();
+  const style = {
```

Comment:
Same here with using the theme from `sx` and putting the styles on the components they belong to!